New-bornhom-sandbox

```diff
@@ -0,0 +1,7 @@
+This is a new sandbox for the new players
+
+Architecture:
+The dimensions of the sandbox are 5 meters by 5 meters.
+
```

Comment:
@Lezago is this code okey for merging or do you have more things to do.

Reply:
@KGunnarsson - I would like to crowdsource ideas for toys for the sandbox as well. do you have any ideas what kind of toys we could add before launch?